content/en/Integrations/IntegrationBuilder/How to Guides/jira-cloud-migration.md

```diff
@@ -10,9 +10,9 @@ description: >
 
 This guide is designed to help organizations transition from our native **Jira Cloud** integration to our highly customizable [**Integration Builder**](/integrations/integrationbuilder/).
 
-> ⚠️ This migration guide is not applicable if your organization uses **Jira Server** or **Jira Data Center**.
+> **⚠️** This migration guide is not applicable if your organization uses **Jira Server** or **Jira Data Center**.
 
-> ⚠️ The Integration Builder is currently in Open Beta status. We do not recommend that all users migrate to the Integration Builder at this time. We recommend that you migrate to the Integration Builder if our current generally-available Jira integration does not meet your needs.
+> **⚠️** The Integration Builder is currently in Open Beta status. We do not recommend that all users migrate to the Integration Builder at this time. We recommend that you migrate to the Integration Builder if our current generally-available Jira integration does not meet your needs.
```

Comment:
> **⚠️** The Integration Builder is currently in Open Beta status. We do not recommend

We can remove this line now that the IB is GA

Reply:
Removed in https://github.com/cobalthq/cobalt-product-public-docs/pull/558/commits/a21cbd3cd25ed1dc995749ca64073cdaea75bd9b.